.travis.yml

```diff
@@ -2,7 +2,7 @@ language: python
 dist: xenial
 sudo: required
 python:
- - "3.7"
```

Comment:
Do we just want to add a Travis CI build matrix and still support 3.7 as well as 3.8 to be backwards compatible to consumers?